stackoverflow-dark.css

```diff
@@ -163,11 +163,16 @@
     border-color: #555 !important;
     color: #ddd !important;
   }
+  
+  /* === herobox when not logged in === */
+#herobox.alternate {
+    background-color: rgba(0, 0, 0, 0.2) !important;
+}
 
   #header #topbar, .revision-comment, .date-span, .post-tag, .form-submit input, .preferred-login,
   .mdhelp, .wmd-help-button.active-help, .date_brick, .more-info, .review-stats-current-user,
-  .review-stats-count-current-user {
-    background: #222 !important;
+  .review-stats-count-current-user, .topbar.newheader, .topbar-menu-links, .network-items>a, .icon-help, .icon-help:hover, a:nth-child(2):hover  {
+    background-color: #222 !important;
```

Comment:
When background:
![image](https://cloud.githubusercontent.com/assets/16362320/21081916/b6a57d62-bfd0-11e6-8858-b3b583d929f3.png)

When background-color:
![image](https://cloud.githubusercontent.com/assets/16362320/21081925/e2c3e2da-bfd0-11e6-87c3-d91e65298181.png)

Sorry about the random search text :P.